WcaOnRails/app/models/person.rb

```diff
@@ -280,6 +280,11 @@ def serializable_hash(options = nil)
       id: self.wca_id,
     )
 
+    if options[:option_name] == :can_admin_results
+      json[:dob] = dob.to_s
+      json[:incorrect_wca_id_claim_count] = incorrect_wca_id_claim_count
+    end
```

Comment:
This is not the correct way to handle serialization for specific controllers.
The controller calling the serialization method should add the *specific names* of the parameters it requires in the `only` or `methods` keys similar to `DEFAULT_SERIALIZE_OPTIONS` above.

Reply:
See `api/v0/competitions_controller#show` for inspiration